### PR TITLE
Cards grid

### PR DESCRIPTION
**Pull request by Nora-Sarrawi**

### CODE REVIEW

File: src/components/cardsContainer/CardsContainer.module.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.container {
```
Comment (mohammadAlawnah) [inline src/components/cardsContainer/CardsContainer.module.css:1]: You cannot modify the Container class. Please give another class name and make modifications to it

File: src/components/card/Card.module.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.container {
```
Comment (mohammadAlawnah) [inline src/components/card/Card.module.css:1]: You cannot modify the Container class. Please give another class name and make modifications to it

File: src/components/card/Card.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import classes from "./Card.module.css";
+
+const Card = ({ icon, image, title, details }) => {
+  return (
+    <div className={classes.container}>
```
Comment (mohammadAlawnah) [inline src/components/card/Card.js:6]: They should all be placed inside dev and given a container class

File: src/components/cardsContainer/CardsContainer.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Card from "../card/Card";
+import classes from "./CardsContainer.module.css";
+import Features from "../featuresWithButton/Features";
+const CardsContainer = () => {
+  return (
+    <div className={`${classes.container} px-sm-2 px-md-3 px-lg-5`}>
```
Comment (mohammadAlawnah) [inline src/components/cardsContainer/CardsContainer.js:7]: Give the parent margin top

**Review by mohammadAlawnah — COMMENTED** [review]
Please pay attention to the instructions and repeat the pull request again

File: src/components/cardsContainer/CardsContainer.module.css (hunk 1 above)
Comment (Nora-Sarrawi) [reply src/components/cardsContainer/CardsContainer.module.css:1]: ok, thank you for noticing

File: src/components/card/Card.js (hunk 3 above)
Comment (Nora-Sarrawi) [reply src/components/card/Card.js:6]: i will do that

File: src/components/cardsContainer/CardsContainer.js (hunk 4 above)
Comment (Nora-Sarrawi) [reply src/components/cardsContainer/CardsContainer.js:7]: i will update it now

**Pull request closed without merging**